### PR TITLE
Created an even newer file

### PR DESCRIPTION
**Pull request by adeelkazmi**
Change-Id: Ic23a44324380b80dcfc35257b0a3f81244c1c8c6

### CODE REVIEW

File: NewerFile.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Created a new file.
```
Comment (DavidIanSteele) [inline NewerFile.txt]: Amend for the win?

File: NewerFile.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+Now amending.
+And again.
```
Comment (adeelkazmi) [inline NewerFile.txt]: Why so many? !fix

Comment (DavidIanSteele) [inline NewerFile.txt]: !resolved

File: NewerFile.txt (hunk 1 above)
Comment (DavidIanSteele) [inline NewerFile.txt]: !completed

**Pull request merged**